netdata-installer.sh

```diff
@@ -443,6 +443,12 @@ if [ -d "${NETDATA_PREFIX}/etc/netdata" ]; then
 	done
 fi
 
+# default directory for certificates
+if [ ! -d "${NETDATA_PREFIX}/etc/netdata/ssl" ]; then
+    run mkdir "${NETDATA_PREFIX}/etc/netdata/ssl"
+    run chown -R "${ROOT_USER}:${NETDATA_GROUP}" "${NETDATA_PREFIX}/etc/netdata/ssl"
+fi
```

Comment:
On clean install i get

```bash
[/home/ilyam/src/test/netdata]# mkdir /opt/netdata/etc/netdata/ssl 
mkdir: cannot create directory '/opt/netdata/etc/netdata/ssl': No such file or directory
 FAILED   

[/home/ilyam/src/test/netdata]# chown -R : /opt/netdata/etc/netdata/ssl 
chown: cannot access '/opt/netdata/etc/netdata/ssl': No such file or directory
 FAILED   
```

Steps to reproduce:
 - remove netdata
 - install netdata

`ssl` directory is not created during install

Reply:
Thanks! I assumed that it exists before, so I used now "mkdir -p" to fix this. Thanks!

Reply:
Get rid of this section completely. Just add `"ssl"` just like `"custom-plugins.d"` was added in #6080 and #6089 (two different files, changing just one line containing the list of the subdirectories).

Reply:
All right, I saw the lines and I already changed!

Reply:
Push the changes so we can resolve this plz.

Reply:
I mentioned two files. `packaging/makeself/install-or-update.sh` also needs to be updated.

Reply:
I updated both with the newest pull, and before this I also finish the redirect using Header instead a HTML page, but I kept the HTML page that redirects to simplify the life of a users that want to use the API to request something, with the page in hand they do not need to see the headers.

Reply:
The comment on the redirect shouldn't be here, but install-or-update.sh is updated.